docs/config.md

```diff
@@ -10,6 +10,53 @@ We have the follow syntax rules:
 
 Our goal is to have an distinct meaning for each column. This way if we want to extend the rules we can add additional columns.
 
+## Client config `~/.opk/config.yml`
+
+The config file for the client is saved in `~/.opk/config.yml`.
+It configures OpenID Providers that the user can log in with and only details of the client.
```

Comment:
"and only details of the client."

I'm not sure what this second half of the sentence means.

Reply:
Fixed